CHANGELOG.md

```
@@ -1,102 +1,100 @@
+#### 9.5.1 - June 2024
   * (PW-1913) Added IBAN validation for Egypt local account structure
   * Restore deprecated method in MT210 class
 
-#### 10.2.0 - May 2024
+#### 9.5.0 - May 2024
   * SWIFT Standard release update 2024 (live 16 November 2025)
   * Yearly revision of deprecation phase (see https://dev.prowidesoftware.com/SRU2024/getting-started/deprecation/)
-  * Dependency update: commons-lang3 -> 3.14.0
-  * Dependency update: gson -> 2.11.0
+  * Dependency update: commons-lang3 -> 3.14.0'
+  * Dependency update: gson -> 2.11.0'
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)